toolchain/check/handle_class.cpp

```diff
@@ -512,6 +512,15 @@ auto HandleParseNode(Context& context, Parse::BaseDeclId node_id) -> bool {
     return true;
   }
 
+  if (!context.struct_type_fields_stack().PeekArray().empty()) {
+    CARBON_DIAGNOSTIC(BaseDeclAfterFieldDecl, Error,
+                      "`{0}` declaration appears after field declaration(s)",
```

Comment:
```suggestion
                      "`{0}` declaration must come before field declarations",
```

Suggesting different phrasing to avoid the `(s)`

Reply:
Alternately:

```suggestion
                      "`{0}` declaration must be the first field declaration",
```

Reply:
Hmm, I don't really think of the base as a field - but perhaps other folks/carbon wants to/does?

I went with the first option, but with "appear before" rather than "come before" because that sounded better to me and we have "appear before" and no "come before" (though we also have one "come after" and no "appear after")